spec/models/solidus_bolt/payment_source_spec.rb

```diff
@@ -14,8 +14,7 @@
       'card_id',
       'create_bolt_account',
       'created_at',
-      'updated_at',
-      'user_id'
+      'updated_at'
```

Comment:
I don't see the purpose of the spec `Check Model Integrity`. It looks to me that `has correct table name` and `has correct attributes` are not related to our code, but implicit tested by ActiveRecord.
I think we should remove them in favour of just the presence validator for the payment_method.

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/89/commits/d511d4222ae067c87c9ef303d220e1f64ca0e146